### DISCUSSION TITLE
About PNM stitch function

### DISCUSSION THREAD

**Discussion by likeduck** [PMEAL/OpenPNM#2805] [Q&A]
Hi! OpenPNM teams!
  I want to stitch two cubic PNMs, they are in same size(from PoreSpy-SNOW). I move the second PNM 1.1 times length of its side, and catch the Pnetwork & P_donor by 0.05*length of its side, respectively. But it give a feedback information like this, do you know this situation? Could you please give me some idea?
![Figure_1](https://github.com/PMEAL/OpenPNM/assets/117707506/5c3cf036-6c58-4a41-9d2b-ad41b17ce214)

![捕获](https://github.com/PMEAL/OpenPNM/assets/117707506/b28c629d-b68b-4907-b867-ab2ba32a04ad)

**Comment by jgostick** [comment]
That error is becase the project is not being found.  It can be caused by several reasons and it's difficult to know which one is the problem here.  The following code snippet works for me, so maybe use this as a guide?

```
pn = op.network.Cubic(shape=[25, 25, 1], spacing=1e-4)
pn2 = op.network.Cubic([5, 5, 1], spacing=5e-4)
pn2['pore.coords'] += np.array([35e-4, 0, 0])
op.topotools.stitch(pn, pn2, P_network=pn.pores('right'), P_donor=pn2.pores('left'))
op.visualization.plot_connections(pn)
```

**Reply by likeduck** [reply]
Thank you for your prompt reply! It's helpful for my current situation. After successfully debugging for a while, this issue resurfaced. I'll need to delve into it further and then discuss it here.🤯
